### DISCUSSION TITLE
ScriptReference/SceneManagement.EditorSceneManager-newSceneCreated

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#586] [Notes]
# ScriptReference/SceneManagement.EditorSceneManager-newSceneCreated

https://docs.unity3d.com/ScriptReference/SceneManagement.EditorSceneManager-newSceneCreated.html

**Comment by electronics45** [comment]
Note that this event is only fired when calling [Editor]SceneManager.NewScene(), or by selecting "Add New Scene" in the hierarchy's breadcrumb menu.  It is NOT called for new scenes created in the project window even after opening them.
